envoy.yaml

```diff
@@ -0,0 +1,52 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 127.0.0.1, port_value: 9901 }
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        protocol: TCP
+        address: 0.0.0.0
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        stat_prefix: ingress_http
+        config: 
+          stat_prefix: ingress_http
+          codec_type: AUTO
+          route_config:
+            name: local_route
+            virtual_hosts:
+            # the name of the virtual host
+            - name: local_service
+            # A list of domains (host/authority header) that will be matched to this virtual host. 
+            # * will match any host/authority header
+              domains: ["*"]
+              routes:
+              - match:
+                  prefix: "/"
+                route:
+                  cluster: service_ratings
+          http_filters:
+          - name: envoy.router
+  clusters:
+  - name: service_ratings
+    connect_timeout: 1s
+    type: LOGICAL_DNS
+    # Comment out the following line to test on v6 networks
+    dns_lookup_family: V4_ONLY
+    lb_policy: ROUND_ROBIN
```

Comment:
I would be curious about the performance of other (load balancer types)[https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing/load_balancers#arch-overview-load-balancing-types].